althea_types/src/websockets/encryption.rs

```diff
@@ -0,0 +1,221 @@
+//! This file handles encryption and decryption of websocket related data types (all data
+//! sent to ops tools from a router)
+
+use std::fmt::Display;
+
+use crypto_box::{
+    aead::{Aead, AeadCore},
+    PublicKey, SalsaBox, SecretKey,
+};
+use rand::rngs::OsRng;
+
```

Comment:
if you remove the spaces between imports cargofmt will sort them. very minor nit.